### DISCUSSION TITLE
MuJoCo Model / environment creation

### DISCUSSION THREAD

**Discussion by gautam1858** [vikashplus/robohive#136] [General]
Hi I just want to know how to create environment or models ? Is it recommended to use CAD software, design robotic arm and then convert to XML file ? I want to create a environment like having a table where the robotic arm can do certain things like pick and place etc. Is it possible to create xml file directly ? please advice on this.

**Comment by vikashplus** [comment]
I'll recommend directing the Model creation questions to MuJoCo forms. 
- [MuJoCo](https://mujoco.readthedocs.io/en/stable/overview.html)'s documentation is excellent. I'll recommend going through the documentation. I also hear that there's a Blender to MuJoCo exporter that will help you to create models in Blender and export it into Mujoco's importable format. 
- The most common way to start is to take one of the existing models and modify it as per your needs. 

For environments 
- There are quite a few environments in RoboHive quite close to what you are looking for. Please checkout the Arm suite as a starting point.
